### PR TITLE
Issue #73

### PR DESCRIPTION
**Pull request by Altoyyr**
adds config.check_config_sanity method

This method retrieves all config properties in Config 
and tries to load them via the configparser. It
excepts ValueException and prints a useful error-
message via click.

The program terminates if a config-error was found.

### CODE REVIEW

File: twtxt/config.py [hunk 1]
```diff
@@ -110,11 +115,13 @@ def options(self):
 
     @property
     def nick(self):
-        return self.cfg.get("twtxt", "nick", fallback=os.environ.get("USER", "").lower())
```
Comment (timofurrer) [inline twtxt/config.py]: Please, don't do this line break - it really doesn't help at all..

**Comment by Altoyyr** [comment]
I rebased this for easier merging and I am writing a unit-test right now, so please do not merge the request right now.

**Comment by Altoyyr** [comment]
Please check if this implementation to fix issue 73 is ok. I am done right now

**Comment by timofurrer** [comment]
Just to let you know ... you can use the # followed by a issue/PR id to reference it. Like: #73

**Pull request merged**